tests/ci/nightly_gpu.yml

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+variables:
+  test: 'tests/ci/run_pytest.py'
+  maxnodes : 4
+  reponame : 'Recommenders'
+  branch   : 'azure-pipelines-bz'
+  rg       : 'recommender'
+  wsname   : 'RecoWS'
+# GPU
+  vmsize   : 'STANDARD_NC6'
+  dockerproc : 'gpu'
+
+trigger: none
+
+pr:
```

Comment:
Nightly tests should be triggered every day, not when some does a PR (as opposite to unit tests)

Reply:
pr trigger removed from yml.
The nightly trigger will be set in the devops pipeline after the PR is in staging.